common.blocks/user-description/user-description.styl

```diff
@@ -0,0 +1,4 @@
+.user-description
+{
+    color: #1d1d1f;
+}
```

Comment:
http://stylus-lang.com/docs/selectors.html

"Whitespace is significant, so we substitute { and } with an indent, and an outden"